### DISCUSSION TITLE
<React> Allow pass callback to setData

### DISCUSSION THREAD

**Discussion by Pequem** [inertiajs/inertia#734] [Ideas]
Hello,
Can be very usefull if the setData of useForm be can receiver a callback that is called when setdata is complete.

**Comment by reinink** [comment]
Can you give a use case where this would be useful?

**Reply by suxur** [reply]
Unless there is a better way to do this, it'd be useful for a auto-saving form. For example if you have a drag and drop list and want to persist the order in the database, after dropping the you'd setData with the new order and then send that to the back-end to persist it.
